config/routes.rb

```diff
@@ -376,6 +376,11 @@
       get 'contestable_issues(/:benefit_type)', to: 'contestable_issues#index'
     end
     resources :higher_level_reviews, only: %i[create show]
+
+    namespace :notice_of_disagreements do
+      get 'contestable_issues', to: 'contestable_issues#index'
```

Comment:
Are there any specs to cover these new endpoints? Also may need to add swagger doc coverage as well